Dockerfile

```diff
@@ -43,12 +43,20 @@ RUN apt-get update \
 		make \
 		python \
 		rsync \
-		wget \
+		curl \
 	&& rm -rf /var/lib/apt/lists/
 
 COPY package.json package-lock.json /usr/src/app/
 
-RUN JOBS=MAX npm ci --no-optional --unsafe-perm || JOBS=MAX npm install --no-optional --unsafe-perm
+# We first ensure that every architecture has an npm version
+# which can do an npm ci, then we perform the ci using this
+# temporary version
+RUN curl -LOJ https://www.npmjs.com/install.sh && \
+	# This is required to avoid a bug in uid-number
+	# https://github.com/npm/uid-number/issues/7
+	npm config set unsafe-perm true && \
+	npm_install="6.9.0" npm_config_prefix=/tmp sh ./install.sh && \
```

Comment:
Can we somehow share variables in multistage build?
I would extract the version to such a variable if yes, so that there is a single place in the file where it has to be modified if we need to bump it.

Reply:
Done.